### ISSUE TITLE
请问是否有在微信小程序段训练的样例？

### ISSUE THREAD

**Issue by zju-zry** [tensorflow/tfjs-wechat#104]

**Comment by zju-zry** [comment]
`import * as tf from '@tensorflow/tfjs-core'
// import * as tfb from '@tensorflow/tfjs-backend-webgl'

export default() => {

    const a = tf.variable(tf.scalar(Math.random()));
    const b = tf.variable(tf.scalar(Math.random()));
    const c = tf.variable(tf.scalar(Math.random()));
    const d = tf.variable(tf.scalar(Math.random()));
 

    function predict(x) {
        // y = a * x ^ 3 + b * x ^ 2 + c * x + d
        return tf.tidy(() => {
            let y = tf.add(tf.mul(a,tf.pow(x,tf.scalar(3))) // a * x^3
                ,(tf.mul(b,tf.square(x))) // + b * x ^ 2
                ,(tf.mul(c,x)) // + c * x
                ,(d)); // + d
            x.print()
            y.print()
            return y
        });  
    }

    function loss(predictions, labels) {
        // Subtract our labels (actual values) from predictions, square the results, and
        // take the mean.
      const meanSquareError = tf.mean(tf.square(tf.sub(predictions, labels)));
      return meanSquareError;
    }

    function train(xs, ys, numIterations = 75) {

        const learningRate = 0.5;
        const optimizer = tf
            .train
            .sgd(learningRate);

        for (let iter = 0; iter < numIterations; iter++) {
            optimizer.minimize(() => {
                const predsYs = predict(xs);
                return loss(predsYs, ys);
            });
        }
    }

    // 上面这些代码应该是没有问题，但是我不确定应该怎么传入参数
    function generateData(numPoints, coeff, sigma = 0.04) {
        const gety = x =>  coeff.a * x * x * x + coeff.b * x * x + coeff.c * x + coeff.d
        let ans = {xs:[],ys:[]}
        for (let i=0;i<numPoints;i++){
            let x = Math.random()
            let y = gety(x)
            ans.xs.push(x)
            ans.ys.push(y)
        }
        ans.xs =tf.tensor1d(ans.xs)
        ans.ys =tf.tensor1d(ans.ys)
        return ans
    }

    const trueCoefficients = {a: -.8, b: -.2, c: .9, d: .5};
    const trainingData = generateData(100, trueCoefficients);
    // trainingData.xs.print()
    // trainingData.ys.print()
    train(trainingData.xs, trainingData.ys, 70);
    // predict(trainingData.xs)

}`
